ocs_sample_library_preview/Authentication.py

```diff
@@ -1,62 +1,58 @@
-
-from .SdsError import SdsError
-
 import base64
-import configparser
 import hashlib
 from http.server import HTTPServer, BaseHTTPRequestHandler
 import json
-import os
+import requests
 import secrets
 import time
 from urllib.parse import urlparse, parse_qs
 import webbrowser
 
-import requests
+from .SdsError import SdsError
 
 
 class Authentication(object):
 
-    def __init__(self, tenant, url, clientId, clientSecret):
+    def __init__(self, tenant: str, url: str, client_id: str, client_secret: str):
```

Comment:
the __init__'s throughout the existing library directly assign values to the underlying private properties, but the new Assets library assigns them through the publicly exposed setter methods. I like the Assets approach since it moves any potential setter logic into one place.

Since the existing library's private properties are being renamed and explicitly typed in this PR, should they also be changed so the initializers access the setters instead?

Reply:
I think I did that work already for the classes in the SDS and DataViews folders. The only reason it wasn't changed in this file is that there are no public properties for these values on the Authentication object.